packit/api.py

```diff
@@ -2053,6 +2053,46 @@ def watch_copr_build(
             report_func=report_func,
         )
 
+    def run_osh_build(
+        self,
+        chroot: Optional[str] = "fedora-rawhide-x86_64",
+        srpm_path: Optional[Path] = None,
+        upstream_ref: Optional[str] = None,
+        release_suffix: Optional[str] = None,
+        base_srpm: Optional[Path] = None,
+        comment: Optional[str] = "Submitted through PackIt.",
+    ) -> str:
+        """
+        Perform a build through OpenScanHub.
+        """
+        # `osh-cli` requires a kerberos ticket.
+        self.init_kerberos_ticket()
+
+        if not srpm_path:
+            srpm_path = self.create_srpm(
+                upstream_ref=upstream_ref,
+                srpm_dir=self.up.local_project.working_dir,
+                release_suffix=release_suffix,
+            )
+
+        if base_srpm:
+            cmd = [
+                "osh-cli",
+                "version-diff-build",
+                "--srpm=" + str(srpm_path),
+                "--base-srpm=" + base_srpm,
+            ]
+        else:
+            cmd = ["osh-cli", "mock-build", str(srpm_path)]
+
+        cmd.append("--config=" + str(chroot))
+        cmd.append("--nowait")
+        cmd.append("--comment=" + comment)
+
+        cmd_result = self.up.command_handler.run_command(cmd, return_output=True)
```

Comment:
I would probably rather use [this](https://github.com/packit/packit/blob/ce6bd5775ae5deb99260bfcee240febf49151b4e/packit/utils/commands.py#L36) method, as with the current code, this is being run in the upstream repository, which is I think not needed